src/category_theory/discrete_category.lean

```diff
@@ -26,6 +26,10 @@ namespace functor
 { obj := F,
   map := λ X Y f, begin cases f, cases f, cases f, exact 𝟙 (F X) end }
 
+instance of_function_coe {α : Type u₁} {β : Type u₂} [category.{v₂} β] :
+  has_coe (α → β) (discrete α ⥤ β) :=
+{ coe := functor.of_function }
```

Comment:
This looks like a dangerous coercion to me, especially if we need to solve `has_coe (α → β) ?m`.

Reply:
I agree; removed.